src/plugins/annotation/annotation-info.js

```diff
@@ -59,7 +64,12 @@ class AnnotationInfo {
   }
 
   getCdnResources(content) {
-    return this.gfm.extractCdnResources(content.text);
+    const cdnResources = [];
+
+    cdnResources.push(...this.gfm.extractCdnResources(content.title));
+    cdnResources.push(...this.gfm.extractCdnResources(content.text));
```

Comment:
Oh this was missing too.. Which made me realize we should probably do a CDN resource regeneration and document regeneration on ELMU (before final import to OMA) and on OMA (after this change is deployed). 
We have had a few more scenarios in the past, so would be good to refresh the data 🤔

Reply:
I thought the same. Although it didn't render markdown before, so probably nobody put markdown there until now. Nevertheless, let's recreate them after releasing.